ImageCoordinates coordinatesTopLeft
ConeColor coneType
uint16 boxWidth
uint16 boxHeight
